modules.d/35network-wicked/module-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    local _program
+
+    require_binaries wicked || return 1
+
+    # do not add this module by default
+    return 255
+}
+
+# called by dracut
+depends() {
+    echo systemd dbus
+    return 0
+}
+
+# called by dracut
+installkernel() {
+    return 0
+}
+
+# called by dracut
+install() {
+    inst_hook cmdline 99 "$moddir/wicked-config.sh"
+
+    # Seems to not execute if in initqueue/settled
+    inst_hook pre-mount 99 "$moddir/wicked-run.sh"
+
+    inst_dir /etc/wicked/extensions
+    inst_dir /usr/share/wicked/schema
+    inst_dir /usr/lib/wicked/bin
+    inst_dir /var/lib/wicked
+
+    inst_multiple /etc/wicked/*.xml
+    inst_multiple /etc/wicked/extensions/*
+    inst_multiple /etc/dbus-1/system.d/org.opensuse.Network*
+    inst_multiple /usr/share/wicked/schema/*
+    inst_multiple /usr/lib/wicked/bin/*
```

Comment:
I've noticed that due to a macro change in our build service, now the wickedd-* files could land either on `usr/lib/wicked/bin/` or on `/usr/libexec/wicked/bin/` depending on when the RPM was build. 
Should we change it for:
   ```
    inst_any /usr/lib/wicked/bin/wickedd-auto4 /usr/libexec/wicked/bin/wickedd-auto4
    inst_any /usr/lib/wicked/bin/wickedd-dhcp4 /usr/libexec/wicked/bin/wickedd-dhcp4
    inst_any /usr/lib/wicked/bin/wickedd-dhcp6 /usr/libexec/wicked/bin/wickedd-dhcp6
```
?

Reply:
I am now accounting for both contingencies.